pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -566,8 +567,21 @@ func (m *MigCluster) GetInternalRegistryPath(c k8sclient.Client) (string, error)
 	}
 }
 
+func (m *MigCluster) SetRegistryPath(c k8sclient.Client) error {
+	newRegistryPath, err := m.GetRegistryPath(c)
+	if err != nil {
+		return err
+	}
+	m.Status.RegistryPath = newRegistryPath
```

Comment:
@djwhatle We're not using this field on the controller. We explicitly call m.GetRegistryPath() when we need it. I added this field at the request of the UI team so that they could display to the user the registry path that would be used.

Reply:
Gotcha. Sounds good then.